tutorials/quickstart-node.md

```diff
@@ -0,0 +1,497 @@
+app.listen(port, () => console.log(`Example app listening at http://localhost:${port}`))
+```
+
+You can test your simple application by running the following from your
+command line and using your browser to view http://localhost:3000:
+
+```bash
+node index.js
+```
+
+You should get a "Hello, World" greeting.
+
+#### Step 2: Configure the TimescaleDB database using Sequelize
+
+Locate your TimescaleDB credentials in order to connect to your TimescaleDB instance.
+
+You’ll need the following credentials:
+
+* password
+* username
+* host URL
+* port
+* database name
+
+Now, let's add Sequelize to our project by first installing it (and its command 
+line interface) and the packages for PostgreSQL from the command line:
+
+```bash
+npm install sequelize sequelize-cli pg pg-hstore
+```
+
+Now let's go back to our `index.js` file and require Sequelize in our application. 
+You'll need your TimescaleDB credentials in order to build the connection URL 
+as well. Once you have that information, add the following to `index.js`, below 
+the other `const` statements:
+
+```javascript
+const Sequelize = require('sequelize')
+const sequelize = new Sequelize('postgres://user:pass@example.com:5432/dbname',
+    {
+        dialect: 'postgres',
+        protocol: 'postgres',
+        dialectOptions: {
+            ssl: {
+                require: true,
+                rejectUnauthorized: false
+            }
+        }
+    })
+```
+
+>:WARNING: Note the settings in `dialectOptions`. These are critical in connecting to a TimescaleDB instance via SSL. 
+
+We can test this connection by adding the following to `index.js` after the `app.get` statement:
+
+```javascript
+sequelize.authenticate().then(() => {
+    console.log('Connection has been established successfully.');
+}).catch(err => {
+    console.error('Unable to connect to the database:', err);
+});
+```
+
+Once again, start the application on the command line:
+
+```bash
+node index.js
+```
+
+And you should get the following results:
+
+```bash
+Example app listening at http://localhost:3000
+Executing (default): SELECT 1+1 AS result
+Connection has been established successfully.
+```
+
+### Create a relational table [](create_table)
+
+#### Step 1: Add TimescaleDB to your Node configuration
+
+Now that we have a successful connection to the `defaultdb` database, we 
+can build out our first database and model.
+
+Let's initialize Sequelize and create the necessary configuration files for our
+project. From the command line, type the following:
+
+```bash
+npx sequelize init
+```
+
+This will create a `config/config.json` file in your project. We will need to
+modify it with the connection details we tested earlier. For the remainder of
+this application, we'll use a database called `node_test`. Here's a full example
+file. Again, note the `dialectOptions`.
+
+```json
+{
+  "development": {
+    "username": "tsdbadmin",
+    "password": "your_password",
```

Comment:
All of the parameters that should be filled out are usually surrounded by `[]`. (`[your_password]`)